Library/Homebrew/cask/cask_file.rb

```diff
@@ -5,7 +5,7 @@
 require "searchable"
 
 module Hbc
-  class Cask
+  class CaskFile
```

Comment:
👎  on this, since this isn't a file.

Reply:
@reitermarkus Suggested alternative given my comment to `@amyspark` above?

Reply:
The only option I see is making `Cask` (formerly `Hbc`) a class rather than a module and defining all this on there.